packages/teams-js/src/public/meeting.ts

```diff
@@ -396,7 +396,7 @@ export namespace meeting {
   /**
    * @hidden
    * Allows an app to get the authentication token for the anonymous or guest user in the meeting
```

Comment:
This line still says "in the meeting", but line 399 says that this works for side panel, meeting stage, and task module. That seems like a contradiction.